admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.ts

```diff
@@ -28,7 +28,7 @@
 export default {
   mainDiv: '#order-view-page',
   orderPaymentDetailsBtn: '.js-payment-details-btn',
-  orderPaymentFormAmountInput: '#order_payment_amount',
+  orderPaymentFormAmountInput: '#order_payment_amount_currency_amount',
```

Comment:
order_payment_amount_currency_amount => don't understand what it means ... without the last amount, I understand it's an order amount with currency .... but the last amount makes some blur in my mind

Reply:
The id is autogenerated from the form type `amount`. 

cf OrderPaymentType AmountCurrency

Reply:
it's sad :-(